### ISSUE TITLE
User comments not being displayed on Guardian articles

### ISSUE THREAD

**Issue by Objectivite** [MrAlex94/Waterfox#2742]
### What happened?

Waterfox 4.1.3.1

https://www.theguardian.com/commentisfree/2022/jul/04/keir-starmer-labour-brexit-plan-mouse-policy-afraid

Impossible have the comments displayed on that page, and many other opinion articles.

This all works on Firefox, Edge, and Chrome, however.

O.

### Reproducible?

- [X] I have tested this on Firefox.

### Version

G4

### What platform are you seeing the problem on?

Windows

**Comment by adamp01** [comment]
Hi @Objectivite - Am I missing something here, as comments seem to display fine on my end?
![image](https://user-images.githubusercontent.com/47503375/177557124-4fc16585-0400-4e51-aea7-f3fa7e18980a.png)

**Comment by Objectivite** [comment]
Hi Adam,

When I reported this I could not get that page up in Waterfox on my PC, laptop, or a second PC.  In fact I couldn't read reader comments on any article, not just that one.  I even cleared the cache at the time.

I then tried Firefox and it worked, Microsoft Edge and that worked, so I submitted the bug.

I have just gone onto The Guardian and you are right, reader comments are accessible, so go figure.

You might as well close this one - Thanks for looking at it.

O.
